RFC/src/Glossary.md

```diff
@@ -161,6 +170,11 @@ A Mining Server is responsible for constructing new blocks by bundling transacti
 
 A Mining Worker is responsible for performing Proof-of-Work tasks received from its parent [Mining Server].
 
+## Node id
```

Comment:
The node ids primary purpose has always been to specify the location of an entity in the network, but it might have secondary uses such for algorithmic committee selection etc. In the Tari communication network all Validator Nodes, Base Nodes, Wallets and Token Wallets will have a node id, but only Validator nodes obtain them by registering on the Base Layer.

Reply:
So this is the offending description, excluding the rest: "_A node id is a unique identifier that specifies the location of a Consensus Node or Consensus Client in the Tari communication network._"

Reply:
Understood now, thanks, that VNs borrow the node ID from the CN. Perfect